src/main/scala/org/broadinstitute/dsde/workbench/sam/db/tables/TosTable.scala

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.dsde.workbench.sam.db.tables
+
+import org.broadinstitute.dsde.workbench.model._
+import org.broadinstitute.dsde.workbench.sam.db.SamTypeBinders
+import org.broadinstitute.dsde.workbench.sam.model.SamUserTos
+import scalikejdbc._
+
+import java.time.Instant
+
+final case class TosRecord(
+    samUserId: WorkbenchUserId,
+    version: String,
+    action: String,
+    createdAt: Instant
+)
+
+object TosTable extends SQLSyntaxSupportWithDefaultSamDB[TosRecord] {
+  val ACCEPT = "ACCEPT"
+  val REJECT = "REJECT"
```

Comment:
Maybe theres a better way/place to put this? Id love to have an enum but that can be a pain in scala.